docs/hwids.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Hardware IDs
+---
+
+## Introduction
+
+Hardware IDs are used by fwupd to identify specific hardware.
+This is useful as the device-specific identifiers may be the same for different firmware streams.
+Each hardware ID has varying levels of specificity for the hardware, for instance matching only the
+system OEM, or matching up to 8 fields including the system BIOS version.
+
+For instance, Dell and Lenovo might ship a wireless broadband modem with the same chip vendor and
+product IDs of `USB\VID_413C&PID_81D7` and although the two OEMs share the same internal device,
+the firmware may be different.
+To cover this case fwupd allows adding `hardware` requirements that mean we can deploy firmware that
+targets `USB\VID_413C&PID_81D7`, but *only for Dell* or *only for Lenovo* systems.
+
+Microsoft calls these values "CHIDs" and they are generated on Windows using the `ComputerHardwareIds.exe`
+binary which can be found [here](https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/computerhardwareids).
+The list of Windows 10 CHIDs are:
```

Comment:
Although some are only valid for Win 10 or newer, suggest dropping the 10.  If we need to disambiguate less then 10 that's not needed in this context to me.

Reply:
Fixed, agreed.